<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaborFilterDialog</class>
 <widget class="QDialog" name="GaborFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnDismiss">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dismiss</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProcess">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../ijresources.qrc">
        <normaloff>:/ijicons32/ijplay32</normaloff>:/ijicons32/ijplay32</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAzimuth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInitialFrequency">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.001</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>steps:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinNumberOfFrequencySteps">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>final:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Spatial frequency:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>initial:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFinalFrequency">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtThreshold">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1000.0</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frm3DDisplay">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ijresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnProcess</sender>
   <signal>clicked()</signal>
   <receiver>GaborFilterDialog</receiver>
   <slot>onPerformGaborFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>GaborFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPerformGaborFilter()</slot>
 </slots>
</ui>
